code/modules/mob/living/carbon/superior_animal/human/subtype/eventwolf/eventwolf

```diff
@@ -0,0 +1,51 @@
+/mob/living/carbon/superior_animal/human/eventwolf
+	name = "Void Wolf Marauder"
```

Comment:
I'd suggest using a different name since Marauder is already an in-use term referring to specific, elite voidwolves.

Reply:
Switched to 'Juggernaut'